packages/caliper-application/benchmark/simple/transfer.js

```diff
@@ -0,0 +1,47 @@
+/**
```

Comment:
Please use a license header like in https://github.com/hyperledger/caliper/blob/master/packages/caliper-fabric-ccp/lib/fabric.js